games-fps/serioussam-tfe-data/serioussam-tfe-data-1.5-r1.ebuild

```diff
@@ -0,0 +1,70 @@
+# Copyright 1999-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit cdrom unpacker
+
+PATCH_PREFIX="Patch_1.05_FE"
+# Game name
+GN="serioussam"
+
+DESCRIPTION="Croteam's Serious Sam Classic The First Encounter ... the data files"
+HOMEPAGE="https://www.croteam.com/
+	https://store.steampowered.com/app/41050/Serious_Sam_Classic_The_First_Encounter/"
+SRC_URI="https://github.com/tx00100xt/serioussam-mods/raw/main/Patches/${PATCH_PREFIX}.tar.xz"
+S="${WORKDIR}"
+
+LICENSE="all-rights-reserved"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+RESTRICT="bindist"
+
+pkg_setup() {
+	cdrom_get_cds "Install/1_00_music.gro"
+}
+
+src_unpack() {
+	mkdir Levels Mods || die "failed create dirs"
+	cat "${DISTDIR}/${PATCH_PREFIX}.tar.xz" > "${PATCH_PREFIX}.tar.xz" \
+		|| die "failed to copy patch 1.05"
+	unpack "${WORKDIR}/${PATCH_PREFIX}.tar.xz"
+}
+
+src_install() {
+	local dir="/usr/share/${GN}"
+
+	einfo "Copying from ${CDROM_ROOT}"
+	insinto "${dir}"
+	doins -r "${CDROM_ROOT}"/Install/*
+
+	mv "${WORKDIR}"/*.gro "${D}/${dir}" || die "failed to moved patch 1.05"
+	mv "${D}/${dir}"/Scripts/PersistentSymbols.ini "${WORKDIR}" \
```

Comment:
This should be "${ED}", not "${D}", as ${dir} does not include ${EPREFIX}. Same below too.

Or just prepend it to dir.